### PR TITLE
Pilk

### PR DESCRIPTION
**Pull request by GermanSanta**
:cl:
rscadd: Adds Pilk created by combining space pe... cola and milk! Heals bones in cryo. Would not suggest taking in high doses.
pilk

### CODE REVIEW

File: vgstation13.dme [hunk 1]
```diff
@@ -1516,9 +1516,9 @@
 #include "code\modules\components\ai\conversation.dm"
 #include "code\modules\components\ai\def_zone_handler.dm"
 #include "code\modules\components\ai\door_opener.dm"
+#include "code\modules\components\ai\hearing.dm"
```
Comment (Exxion) [inline vgstation13.dme]: Please revert this change. It was presumably accidental and would probably cause problems with Git later.

EDIT: I should specify that you should move this line back to where it used to be, not delete it

**Comment by Exxion** [comment]
Looks like it heals bones always, not just in cryo.

File: vgstation13.dme (hunk 1 above)
Comment (GermanSanta) [reply vgstation13.dme]: Fixed. Didn't even know the dme file was edittable.

**Comment by GermanSanta** [comment]
> Looks like it heals bones always, not just in cryo.

I'll probably keep it that way for skeletons. Pilk is Milk after all

**Comment by SonixApache** [comment]
this is dumb

**Comment by Eneocho** [comment]
Imagine needing specialized chems for strong bones -This post was approved by the skeleton metaclub.

**Comment by Viciousaurus** [comment]
Another step closer towards Cryotube-only medbay

**Comment by Eneocho** [comment]
Due to the way cryo injects reagents, anyone getting in will most likely get the extreme dizziness (and possibly the overdose) from high doses. Also, seems you didn't add the "heals bones if cold" part.

**Comment by GermanSanta** [comment]
This is literally just milk that makes you sick if you drink too much

**Comment by ShiftyRail** [comment]
updoots are updoots

**Comment by djkramer123** [comment]
powerful permabanned shitter pr

**Pull request closed without merging**